Library/Homebrew/cask/lib/hbc/container/cab.rb

```diff
@@ -8,9 +8,10 @@ class Cab < Base
       def self.me?(criteria)
         cabextract = which("cabextract")
 
+        # Ignores lines containing WARNING, but fails on other errors.
         criteria.magic_number(/^(MSCF|MZ)/n) &&
           !cabextract.nil? &&
-          criteria.command.run(cabextract, args: ["-t", "--", criteria.path.to_s]).stderr.empty?
+          criteria.command.run(cabextract, args: ["-t", "--", criteria.path.to_s]).stderr.split("\n").reject { |l| l.include? "WARNING" }.empty?
```

Comment:
The exit status is still `0` when there are warnings, right? If so, I think we can rely on the `exit_status` instead of checking the `stderr` output.

Also – highly unlikely to happen – this would fail if the file path contained “WARNING” and the exit code was non-zero.